common/pom.xml

```diff
@@ -128,6 +128,11 @@
             <version>${spring.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-webmvc</artifactId>
+            <version>4.0.3.RELEASE</version>
+        </dependency>
```

Comment:
needed for
import org.springframework.web.servlet.ModelAndView;
import org.springframework.web.servlet.handler.HandlerInterceptorAdapter;
in common/src/main/java/tds/common/web/interceptors/EventLoggerInterceptor.java

Will catch up with you on added the spring boot dependency.